pulp_smash/resources/__init__.py

```diff
@@ -0,0 +1,4 @@
+# coding=utf-8
+"""Modules for easy manipulation with pulp resources like users, repositories
+etc."""
```

Comment:
The `pulp_smash.resources.platform` and `pulp_smash.resources.platform.api_v2` modules have identical docstrings. As a result, the purpose of each module is not clear. Other developers will be asking themselves questions like "where does my new helper code go in this hierarchy of modules," and I'd like it if they can answer such questions by reading the docstrings for these modules.